app/models/journeys/nics/NICsJourneyAnswers.scala

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.journeys.nics
+
+import controllers.standard
+import models.common.BusinessId
+import models.database.UserAnswers
+import pages.nics._
+import play.api.libs.json.{Format, Json}
+import play.api.mvc.Result
+import play.api.mvc.Results.Redirect
+
+case class NICsJourneyAnswers(class2Answers: Option[NICsClass2Answers], class4Answers: Option[NICsClass4Answers])
+
+object NICsJourneyAnswers {
+  implicit val formats: Format[NICsJourneyAnswers] = Json.format[NICsJourneyAnswers]
+
+  def buildFromUserAnswers(userAnswers: UserAnswers): Either[Result, NICsJourneyAnswers] = {
```

Comment:
I would add an unit test for this. Tests are also live documentation, and here we decided to fail when we have both class2 and class4 which would be nice to cover. 

Also this test would be nice safety net if we decide to mess with Class2 and Class4 mongo fields.